pkg/instance/instance.go

```diff
@@ -123,6 +124,11 @@ func (i *Instance) FS() afero.Fs {
 	return i.storage
 }
 
+// JobsBroker returns the jobs broker associated with the instance
+func (i *Instance) JobsBroker() (jobs.Broker, error) {
```

Comment:
I don't know.. For the case of the in-memory queue I would tend to agree but for the case of an external queue this getter may initialize a connection with an external system.

Although I agree these are problems we may not want to resolve yet...